build.psm1

```diff
@@ -1571,6 +1571,8 @@ function Start-PSxUnit {
 
```

Comment:
@KirkMunro, your last commit had 32 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

TestDebugJobIPC
```powershell
FAIL A parameter cannot be found that matches parameter name 'BreakAll'.:TestInvokeError
Expected: True
Actual:   False
   at PSTests.Sequential.LocalDebuggerTests.TestDebugJobIPC() in /Users/runner/runners/2.165.0/work/1/s/test/xUnit/csharp/test_Debugger.cs:line 207
```
TestDebugJobWinRM
```powershell
FAIL This parameter set requires WSMan, and no supported WSMan client library was found. WSMan is either not installed or unavailable for this system.:TestInvokeError
Expected: True
Actual:   False
   at PSTests.Sequential.LocalDebuggerTests.TestDebugJobWinRM() in /Users/runner/runners/2.165.0/work/1/s/test/xUnit/csharp/test_Debugger.cs:line 258
```
TestRemoteDebuggerBreak
```powershell
System.AggregateException : One or more errors occurred. (This parameter set requires WSMan, and no supported WSMan client library was found. WSMan is either not installed or unavailable for this system.) (The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.)
---- System.Management.Automation.Remoting.PSRemotingTransportException : This parameter set requires WSMan, and no supported WSMan client library was found. WSMan is either not installed or unavailable for this system.
-------- System.DllNotFoundException : Unable to load shared library 'libpsrpclient' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: dlopen(liblibpsrpclient, 1): image not found
---- System.Management.Automation.CommandNotFoundException : The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

----- Inner Stack Trace #1 (System.Management.Automation.Remoting.PSRemotingTransportException) -----
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager.WSManAPIDataCommon..ctor() in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 2628
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager..ctor(Guid runspacePoolInstanceId, WSManConnectionInfo connectionInfo, PSRemotingCryptoHelper cryptoHelper, String sessionName) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 561
   at System.Management.Automation.Runspaces.WSManConnectionInfo.CreateClientSessionTransportManager(Guid instanceId, String sessionName, PSRemotingCryptoHelper cryptoHelper) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs:line 1087
   at System.Management.Automation.Remoting.ClientRemoteSessionDSHandlerImpl..ctor(ClientRemoteSession session, PSRemotingCryptoHelper cryptoHelper, RunspaceConnectionInfo connectionInfo, URIDirectionReported uriRedirectionHandler) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/remotingprotocolimplementation.cs:line 82
   at System.Management.Automation.Remoting.ClientRemoteSessionImpl..ctor(RemoteRunspacePoolInternal rsPool, URIDirectionReported uriRedirectionHandler) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/clientremotesession.cs:line 214
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler.CreateClientRemoteSession(RemoteRunspacePoolInternal rsPoolInternal) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 473
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler..ctor(RemoteRunspacePoolInternal clientRunspacePool, TypeTable typeTable) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 42
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal.CreateDSHandler(TypeTable typeTable) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 172
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 97
   at System.Management.Automation.Runspaces.RunspacePool..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/hostifaces/RunspacePool.cs:line 606
   at System.Management.Automation.RemoteRunspace..ctor(TypeTable typeTable, RunspaceConnectionInfo connectionInfo, PSHost host, PSPrimitiveDictionary applicationArguments, String name, Int32 id) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/remoterunspace.cs:line 143
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, TypeTable typeTable, PSPrimitiveDictionary applicationArguments, String name) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs:line 557
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, T
```
TestRemoteDebuggerProcessCommand
```powershell
System.AggregateException : One or more errors occurred. (This parameter set requires WSMan, and no supported WSMan client library was found. WSMan is either not installed or unavailable for this system.) (The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.)
---- System.Management.Automation.Remoting.PSRemotingTransportException : This parameter set requires WSMan, and no supported WSMan client library was found. WSMan is either not installed or unavailable for this system.
-------- System.DllNotFoundException : Unable to load shared library 'libpsrpclient' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: dlopen(liblibpsrpclient, 1): image not found
---- System.Management.Automation.CommandNotFoundException : The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

----- Inner Stack Trace #1 (System.Management.Automation.Remoting.PSRemotingTransportException) -----
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager.WSManAPIDataCommon..ctor() in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 2628
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager..ctor(Guid runspacePoolInstanceId, WSManConnectionInfo connectionInfo, PSRemotingCryptoHelper cryptoHelper, String sessionName) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 561
   at System.Management.Automation.Runspaces.WSManConnectionInfo.CreateClientSessionTransportManager(Guid instanceId, String sessionName, PSRemotingCryptoHelper cryptoHelper) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs:line 1087
   at System.Management.Automation.Remoting.ClientRemoteSessionDSHandlerImpl..ctor(ClientRemoteSession session, PSRemotingCryptoHelper cryptoHelper, RunspaceConnectionInfo connectionInfo, URIDirectionReported uriRedirectionHandler) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/remotingprotocolimplementation.cs:line 82
   at System.Management.Automation.Remoting.ClientRemoteSessionImpl..ctor(RemoteRunspacePoolInternal rsPool, URIDirectionReported uriRedirectionHandler) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/clientremotesession.cs:line 214
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler.CreateClientRemoteSession(RemoteRunspacePoolInternal rsPoolInternal) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 473
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler..ctor(RemoteRunspacePoolInternal clientRunspacePool, TypeTable typeTable) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 42
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal.CreateDSHandler(TypeTable typeTable) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 172
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 97
   at System.Management.Automation.Runspaces.RunspacePool..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/hostifaces/RunspacePool.cs:line 606
   at System.Management.Automation.RemoteRunspace..ctor(TypeTable typeTable, RunspaceConnectionInfo connectionInfo, PSHost host, PSPrimitiveDictionary applicationArguments, String name, Int32 id) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/remoterunspace.cs:line 143
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, TypeTable typeTable, PSPrimitiveDictionary applicationArguments, String name) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs:line 557
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, T
```
TestRemoteDebuggerCommandBreakpointUpdated
```powershell
System.AggregateException : One or more errors occurred. (This parameter set requires WSMan, and no supported WSMan client library was found. WSMan is either not installed or unavailable for this system.) (The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.)
---- System.Management.Automation.Remoting.PSRemotingTransportException : This parameter set requires WSMan, and no supported WSMan client library was found. WSMan is either not installed or unavailable for this system.
-------- System.DllNotFoundException : Unable to load shared library 'libpsrpclient' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: dlopen(liblibpsrpclient, 1): image not found
---- System.Management.Automation.CommandNotFoundException : The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

----- Inner Stack Trace #1 (System.Management.Automation.Remoting.PSRemotingTransportException) -----
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager.WSManAPIDataCommon..ctor() in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 2628
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager..ctor(Guid runspacePoolInstanceId, WSManConnectionInfo connectionInfo, PSRemotingCryptoHelper cryptoHelper, String sessionName) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 561
   at System.Management.Automation.Runspaces.WSManConnectionInfo.CreateClientSessionTransportManager(Guid instanceId, String sessionName, PSRemotingCryptoHelper cryptoHelper) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs:line 1087
   at System.Management.Automation.Remoting.ClientRemoteSessionDSHandlerImpl..ctor(ClientRemoteSession session, PSRemotingCryptoHelper cryptoHelper, RunspaceConnectionInfo connectionInfo, URIDirectionReported uriRedirectionHandler) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/remotingprotocolimplementation.cs:line 82
   at System.Management.Automation.Remoting.ClientRemoteSessionImpl..ctor(RemoteRunspacePoolInternal rsPool, URIDirectionReported uriRedirectionHandler) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/clientremotesession.cs:line 214
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler.CreateClientRemoteSession(RemoteRunspacePoolInternal rsPoolInternal) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 473
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler..ctor(RemoteRunspacePoolInternal clientRunspacePool, TypeTable typeTable) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 42
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal.CreateDSHandler(TypeTable typeTable) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 172
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 97
   at System.Management.Automation.Runspaces.RunspacePool..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/hostifaces/RunspacePool.cs:line 606
   at System.Management.Automation.RemoteRunspace..ctor(TypeTable typeTable, RunspaceConnectionInfo connectionInfo, PSHost host, PSPrimitiveDictionary applicationArguments, String name, Int32 id) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/remoting/client/remoterunspace.cs:line 143
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, TypeTable typeTable, PSPrimitiveDictionary applicationArguments, String name) in /Users/runner/runners/2.165.0/work/1/s/src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs:line 557
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, T
```